Flank/flank
# fix: Move shards json to result directory

Fixes #1475 

## Test Plan
> How do we know the code works?

1. Run flank with android or ios test
1. Wait until ```results-directory``` is created
1. Open the results directory and check ```*_shards.json``` exists
1. If   ```disable-results-upload``` not set, check shards file exists in gcloud bucket 
1. Flank shouldn't upload shards file to gcs if ```disable-results-upload``` set


## Checklist

- [X] Documented [(SDD in ticket comment)](https://github.com/Flank/flank/issues/1475#issuecomment-769227112)
